### PR TITLE
Release 8.7.2

### PR DESCRIPTION
**Pull request by alyezz**
* Updates native SDKs

### CODE REVIEW

File: ios/Instabug.framework/Headers/Instabug.h [hunk 1]
```diff
@@ -321,6 +321,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (class, atomic, assign) IBGSDKDebugLogsLevel sdkDebugLogsLevel;
 
+
+#pragma mark - Private Methods for CrossPlatform   /// Will  be removed before the release
```
Comment (HeshamMegid) [inline ios/Instabug.framework/Headers/Instabug.h]: These shouldn't be in the header. Can you please check with iOS about this?

cc: @Kmohamed

File: InstabugSample/__tests__/instabugUtils.spec.js [hunk 2]
```diff
@@ -56,25 +56,25 @@ describe('Test global error handler', () => {
 
     });
 
-    it('should emit event IBGSendUnhandledJSCrash when platform is android and onReportSubmitHandler is set', (done) => {
```
Comment (HeshamMegid) [inline InstabugSample/__tests__/instabugUtils.spec.js]: Why are we removing this?

File: ios/Instabug.framework/Headers/Instabug.h (hunk 1 above)
Comment (Kmohamed) [reply ios/Instabug.framework/Headers/Instabug.h]: @HeshamMegid we will remove it in this release

File: InstabugSample/__tests__/instabugUtils.spec.js (hunk 2 above)
Comment (alyezz) [reply InstabugSample/__tests__/instabugUtils.spec.js]: This is the same test we commented in a diffferent PR, the one that broke even on previous passing builds. Seems to be a problem with the test

**Pull request merged**